server/models/messageModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// Create Schema for Users
+const MessageSchema = new Schema({
+    conversation: {
+        type: Schema.Types.ObjectId,
+        ref: 'conversations',
```

Comment:
This should probably be something that you index (https://mongoosejs.com/docs/2.7.x/docs/indexes.html) for more efficient lookup

Reply:
I will add this to my TODO's as an optimization for the future fixes.